src/actions/GraphDataActions.ts

```diff
@@ -104,7 +112,7 @@ export const GraphDataActions = {
       dispatch(GraphDataActions.getGraphDataStart());
       const duration = graphDuration.value;
       let restParams = { duration: duration + 's', graphType: graphType, injectServiceNodes: injectServiceNodes };
-      if (namespace.name === serverConfig().istioNamespace) {
+      if (namespaces.find(namespace => namespace.name === serverConfig().istioNamespace)) {
```

Comment:
Not sure, but perhaps this needs special handling?
Istio communicates with all nodes with a sidecar. So, when enabling the "istio-system" namespace, it may not make sense to enable/disable the rest of the namespaces.

Reply:
IIUC, do you think we should disallow to show `istio-system` and other namespaces together? or do yo say that I should only pass the `restParams['includeIstio'] = true;` when `istio-system` is the only namespace?

Reply:
I think we should just "do what the user says".  So, if they explicitly select istio-system as one of the namespaces just include it.  So I think the code is OK as is.